docker/Dockerfile.debian-9

```diff
@@ -10,7 +10,7 @@ ENV CCACHE_DIR "/var/spool/ccache"
 ENV CCACHE_COMPRESS 1
 
 ENV CMAKE_DIR "/opt/cmake"
-ENV CMAKE_VERSION "3.17.2"
+ENV CMAKE_VERSION "3.18.0"
```

Comment:
Do we have at least one platform left that's testing with CMake 3.15? Because that's our current mininum requirement, so we should make sure it works with that still (outside of packaging, that's fine).

Reply:
I explicily set a single Docker platform (centos-7) to use the minimum CMake version.